inc/gateways/class-paypal-gateway.php

```
@@ -1631,11 +1799,13 @@ public function get_checkout_details($token = '') {
 
 			$body['pending_payment'] = $pending_payment;
 
-			$custom = explode('|', (string) $body['PAYMENTREQUEST_0_CUSTOM']);
+			$custom = explode('|', (string) wu_get_isset($body, 'PAYMENTREQUEST_0_CUSTOM', ''));
```

PHPMD (2.15.0)

Avoid unused local variables such as '$custom'. (undefined)

(UnusedLocalVariable)